docs/source/deployment/databricks/databricks_ide_development_workflow.md

```diff
@@ -1,6 +1,12 @@
-# Use an IDE, dbx and Databricks Repos to develop a Kedro project
+# Use Databricks Asset Bundles to deploy a Kedro project
 
-This guide demonstrates a workflow for developing Kedro projects on Databricks using your local environment for development, then using dbx and Databricks Repos to sync code for testing on Databricks.
+```{note}
+The `dbx` package is deprecated by Databricks, and dbx workflow documentation is moved to a [new page](./databricks_dbx_workflow.md).
+```
+
+This guide demonstrates a workflow for developing a Kedro Project on Databricks using Databricks Asset Bundles. You will learn how to develop your project using a local environment, then use `kedro-databricks` and Databricks Asset Bundle to package your code for running pipelines on Databricks.
```

Comment:
Maybe add a link here to https://www.databricks.com/resources/demos/tours/data-engineering/databricks-asset-bundles ?

Reply:
addressed